thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -648,7 +654,7 @@ public Response runAdhocEmailConfig(@NotNull @QueryParam("id") Long id) throws E
     if (emailConfiguration == null) {
       throw new IllegalStateException("No emailConfiguraiton for id " + id);
     }
-    detectorHttpUtils.runAdhocEmailConfiguration(String.valueOf(id));
+    alertResourceHttpUtils.runAdhocEmailConfiguration(String.valueOf(id), windowStartIso, windowEndIso);
```

Comment:
Please have the date conversion here itself. Also check if DateTime can be used instead of String in the resource method param itself.